bot/seasons/evergreen/8bitify.py

```diff
@@ -0,0 +1,54 @@
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+class EightBitify(commands.Cog):
+    """Make your avatar 8bit!"""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @staticmethod
+    def pixelate(image: Image) -> Image:
+        """Takes an image and pixelates it"""
+        return image.resize((32, 32)).resize((1024, 1024))
+
+    @staticmethod
+    def quantize(image: Image) -> Image:
+        """Reduces colour palette to 256 colours"""
+        return image.quantize(colors=32)
+
+    @commands.command(name="8bitify")
+    async def eightbit_command(self, ctx: commands.Context) -> None:
+        """Pixelates your avatar and changes the palette to an 8bit one"""
+        async with ctx.typing():
+            image_bytes = await ctx.author.avatar_url.read()
+            avatar = Image.open(BytesIO(image_bytes))
+            avatar = avatar.convert("RGBA").resize((1024, 1024))
+
+            eightbit = self.pixelate(avatar)
+            eightbit = self.quantize(eightbit)
+
+            bufferedio = BytesIO()
+            eightbit.save(bufferedio, format="PNG")
+            bufferedio.seek(0)
+
+            file = discord.File(bufferedio, filename="8bitavatar.png")
+
+            embed = discord.Embed(
+                name="You're 8-bit avatar",
```

Comment:
An `Embed` does not have a `name` field, perhaps you meant `title`?
It should also be `Your` instead of `You're`.
```suggestion
                title="Your 8-bit avatar",
```